GenumKit/Stencil/Filters.swift

```diff
@@ -106,3 +126,83 @@ struct NumFilters {
     return "\(percent)%"
   }
 }
+
+struct ModelParsingFilters {
```

Comment:
Aren't all those too specific? Not sure why we can't implement that in the template instead?